plugins/MidiImport/MidiImport.cpp

```diff
@@ -394,8 +401,8 @@ bool MidiImport::readSMF( TrackContainer* tc )
 			{
 				smfMidiChannel * ch = chs[evt->chan].create( tc, trackName );
 				Alg_note_ptr noteEvt = dynamic_cast<Alg_note_ptr>( evt );
-
-				Note n( noteEvt->get_duration() * ticksPerBeat,
+				double ticks = MidiTime( noteEvt->get_duration() * ticksPerBeat );
```

Comment:
Why is ticks declared as double instead of int? MidiTime only offers an int() cast operator, so casting to double should not give any benefit.